Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTRot-PXI-Controller-Reboot.rst

```diff
@@ -0,0 +1,238 @@
+    to set up the EtherCat and Copley drives before making a connection through the CSC/EUI.
+
+
+
+
+.. _MTRot-PXI-Controller-Reboot-Soft-Reboot:
+
+.. warning::
+
+    Only proceed with :ref:`Soft Reboot <MTRot-PXI-Controller-Reboot-Soft-Reboot>` if EUI/CSC control is not connecting and 
+    the :ref:`restart of the control system <MTRot-PXI-Controller-Reboot-Restart-Control-System>` procedure 
+    proved unsuccessful to regain control.
+
+Soft Reboot
+-----------
+
+
+#.  **Establish an SSH connection to the MTRotator PXI** using the credentials and hostnames found in 
+    the *LSST 1Password MainTel Vault* (See :ref:`above <MTRot-PXI-Controller-Reboot-Restart-Control-System>`).
+
+#.  **Execute the reboot command**: To initiate a soft reboot of the PXI, type in the terminal -
+
+    .. prompt::
+
+        sudo reboot now
+
+#.  **Allow 5-10 minutes for the system to reboot**: This time is necessary for the OS and control system 
+    to configure the EtherCat and Copley drives before attempting a connection through the CSC/EUI. 
+    
+    
+
+
+.. _MTRot-PXI-Controller-Reboot-Hard-Reboot:
+
+Hard Reboot
+-----------
+
+.. warning::
+
+    **Only proceed with a hard reboot, if the EUI control connection remains unsucessful after 
+    a** :ref:`soft reboot <MTRot-PXI-Controller-Reboot-Soft-Reboot>`.
+
+    This method involves cutting power to the PXI and drives and should only be used as a last resort 
+    due to the potential risks.
+
+    If a power shutdown is scheduled, you can proceed until step 2 before the power on.
+
+
+#.  **Login into Utilities cabinet Power Distribution Unit (PDU)**: While in the LSST-WAP, 
+    connect to https://tea-pdu01.cp.lsst.org/ using the credentials stored in the *Operators vault* 
+    of *LSST 1Password* as *PDU Utilities Cabinet*. Click on :guilabel:`Outlets` on the left hand 
+    side menu to open the outlets screen. The description of each outlet can be found here.
+
+    .. figure:: /Simonyi/Non-Standard-Operations/_static/mtrot-controller-pxi-reboot-1.jpeg
+   
+
+#.  **Power Cycle PXI and drives**: To **power** :guilabel:`Off` the system, first power off the PXI, followed by 
+    the drive. For MTRotator (:guilabel:`Camera Rotator` in *PDU*), the PXI is connected to outlet 6, 
+    and drives in outlet 5. 
+
+    .. note::
+
+        **Scheduled Power Off**
+
+        In case a scheduled power off is intended, do not continue with power on, and stop the 
+        procedure here.
+
+
+    When **powering** :guilabel:`On`, activate the drives first, and wait for at least 3 minutes before powering 
+    on the PXI. This delay is **crucial** for the Ethercat application within the PXI to establish a 
+    connection with the Copley drive.
+
+
+#.   **Wait for an additional 5-10 minutes after powering on before using the CSC/EUI**: 
+    This allows time for the OS and control system to configure the necessary drives. 
```

Comment:
The list numbering breaks here ----> This line needs to have an extra tab so that its NOT in the grey box but simply in the same point.